src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -1192,8 +1192,10 @@ pub enum ReplicaOptionName {
     Size,
     /// The `AVAILABILITY ZONE [[=] <id>]` option.
     AvailabilityZone,
-    /// The `STORAGECTL ADDRESS` option.
-    StoragectlAddress,
+    /// The `STORAGE ADDRESSES` option.
+    StorageAddresses,
+    /// The `STORAGECTL ADDRESSES` option.
+    StoragectlAddresses,
```

Comment:
Are these behind unsafe mode? If not we probably want to update the docs with the new syntax.

Reply:
Yes they are behind unsafe mode!